catalog/ui/catalog-ui-search/package.json

```diff
@@ -8,7 +8,8 @@
     "postinstall": "bower install --offline",
     "start": "grunt start",
     "pretest": "node ./node_modules/phantomjs-prebuilt/install.js && webpack --config webpack/config/ci.js",
-    "test": "phantomjs ./node_modules/mocha-phantomjs-core/mocha-phantomjs-core.js ./target/test/index.html"
+    "test": "phantomjs ./node_modules/mocha-phantomjs-core/mocha-phantomjs-core.js ./target/test/index.html",
+    "stat": "webpack --config webpack/config/prod.js --json | webpack-bundle-size-analyzer"
```

Comment:
Did you mean to leave this npm script in?  Do we plan to use this in the future?